package.json

```diff
@@ -2,23 +2,34 @@
   "name": "mathsteps",
   "version": "0.1.6",
   "description": "Step by step math solutions",
-  "main": "index.js",
+  "main": "./build/mathsteps.min.js",
```

Comment:
if `build` is gitignored, will this not work? or will it always build before looking for `main`